Add post about collaboration with sensenet (#485)
SenseNet/sncom-resources: _posts/2021-03-10-5-ways-to-unlock-true-collaboration-with-a-headless-cms.md

---
title: "6 ways to unlock true collaboration with a headless cms"
author: [herflis]
image: "../img/posts/teamwork.png"
tags: [productivity, teamwork, collaboration, saas, headless cms]
redirect_to: https://www.sensenet.com/blog/how-we-post
---

Working on a project in a cross-functional team can be full of pitfalls. I have some tips on how you could use [sensenet](https://profile.sensenet.com/?redirectToLogin) avoiding them and boosting team productivity at the same time.

---

## 1. Versioning and Approval

> Basic building blocks of a collaborative content management process are the different states of content aka. [versions in terms of sensenet](https://docs.sensenet.com/concepts/collaboration/01-versioning). The whole lifecycle of a content from the first draft till the final public version can be tracked using version numbers. Only one setting on a container to switch on versioning and approval and your team will be able to follow if a content should be reviewed, corrected or approved both on the admin surface or with the API.

Reviewer: ```suggestion
Basic building blocks of a collaborative content management process are the different states of content aka. [versions in terms of sensenet](https://docs.sensenet.com/concepts/collaboration/01-versioning). The whole lifecycle of a content from the first draft till the final public version can be tracked using version numbers.  Set versioning and approval on a container and your team will be able to follow if a content should be reviewed, corrected or approved both on the admin surface and with the API.
```

<p align="center">
<img src="/img/posts/collaboration/approval.png" alt="Versioning and approval flow">
</p>

## 2. Sharing

With sensenet you can invite your teammates or customers to share content with them, no matter how far they are from you at the moment. Registered users or groups can join you to work on the same content on various levels (view, edit) and you can share link of a content with external users let them review it or even work together without them having to register. Work together remotely like pros, give your projects a collaborative boost with [content sharing](https://docs.sensenet.com/concepts/collaboration/06-sharing)!

<p align="center">
<img src="/img/posts/collaboration/sharing.png" alt="Sharing">
</p>

## 3. Preview comments

> [Document preview feature](https://docs.sensenet.com/concepts/document-previews) of sensenet is a useful teamwork tool in itself letting you upload your docs and share them with people who don't have the tools to open them. To make it more powerful we have added the possibility to comment any part of the preview images. So this way you could even discuss the design plans with the customer, moving the whole project and all the related content to the same hub.

Reviewer: ```suggestion
[Document preview feature](https://docs.sensenet.com/concepts/document-previews) of sensenet is a useful teamwork tool in itself letting you upload your docs and share them with people who don't have the tools to open them. To make it more powerful we have added the possibility to comment any part of the preview images. This way you could even discuss the design plans with the customer, moving the whole project and all the related content to the same hub.
```

<p align="center">
<img src="/img/posts/collaboration/docviewer.png" alt="Preview comments">
</p>

## 4. Office Online editing

> Why you would use different tools for each work phase when you can write, review, discuss and correct docs directly in the browser? [Office Online editing](https://docs.sensenet.com/concepts/collaboration/03-office-online-editing) is integrated to sensenet making it possible to collaborate on documents as you are used to on your own desktop computer or in google docs. 

Reviewer: ```suggestion
Why would you use different tools for each work phase when you can write, review, discuss and correct docs directly in the browser? [Office Online editing](https://docs.sensenet.com/concepts/collaboration/03-office-online-editing) is integrated to sensenet making it possible to collaborate on documents as you are used to on your own desktop computer or in google docs. 
```
